Class-Code/Elementary-CA/awu/sketch.js

```diff
@@ -1,14 +1,119 @@
-//Your Name(s)
-//Project Name
-//Date
+//Your Name(s) Alex Wu
+//Project Name Elementary CA peoject
+//Date FEB 25
 
-/* A brief description of what the program is */
+/* A brief description of what the program is 
+to mimic the CA operation by javascript*/
+const rules = [0, 0, 0, 1, 1, 0, 1, 1]
+let ca = []
+let cell_size
+let list_length = 201
+let gen_0 = Array(list_length).fill(0)
+gen_0[Math.floor(list_length / 2)] = 1
```

Comment:
this line should really be in setup. its bad practice to do operations outside of a function.